rocky/rocky/views/scheduler.py

```diff
@@ -262,28 +267,31 @@ def run_boefje_for_oois(self, boefje: Boefje, oois: list[OOI]) -> None:
         except SchedulerError as error:
             messages.error(self.request, error.message)
 
-    def convert_recurrence_to_cron_expressions(self, recurrence: str) -> str:
+    def convert_recurrence_to_cron_expressions(self, recurrence: str, start_date_time: datetime) -> str:
         """
-        Because there is no time defined for the start date, we use midnight 00:00 for all expressions.
+        The user defines the start date and time.
         """
 
-        start_date = datetime.now(tz=timezone.utc).date()  # for now, not set by user
-
-        if start_date and recurrence:
-            day = start_date.day
-            month = start_date.month
-            week = start_date.strftime("%w").upper()  # ex. 4
+        if start_date_time and recurrence:
+            day = start_date_time.day
+            month = start_date_time.month
+            week = start_date_time.strftime("%w").upper()  # ex. 4
+            hour = start_date_time.hour
+            minute = start_date_time.minute
 
             cron_expr = {
-                "daily": "0 0 * * *",  # Recurres every day at 00:00
-                "weekly": f"0 0 * * {week}",  # Recurres every week on the {week} at 00:00
-                "yearly": f"0 0 {day} {month} *",  # Recurres every year on the {day} of the {month} at 00:00
+                "daily": f"{minute} {hour} * * *",  # Recurres every day at the selected time
+                "weekly": f"{minute} {hour} * * {week}",  # Recurres every week on the {week} at the selected time
+                "yearly": f"{minute} {hour} {day} {month} *",
+                # Recurres every year on the {day} of the {month} at the selected time
             }
 
             if 28 <= day <= 31:
-                cron_expr["monthly"] = "0 0 28-31 * *"
+                cron_expr["monthly"] = f"{minute} {hour} 28-31 * *"
```

Comment:
We could just set this to 28th for now. To make sure we don't execute on 28,29,30 and 31.

Reply:
I've created a new issue for this: #3810

Reply:
You can use L notation for last day of the month

Reply:
Only *IF* our cron lib understands the L.
And then we should still check if the cosen date is the last day or not.